docs/UserGuide.adoc

```diff
@@ -111,26 +111,26 @@ Forms an English class containing the first, second and fifth contact in the lis
 
 === Setting up an appointment : `set_appointment` `[since v1.3]`
 
-Sets up an appointment with the specified contact. +
-Format: `set_appointment t/TITLE s/START_DATE START_TIME e/END_DATE END_TIME i/INDEX`
+ Sets up an appointment with the specified contact. +
+ Format: `set_appointment t/TITLE s/START_DATE START_TIME e/END_DATE END_TIME i/INDEX`
 
-****
-* This sets an appointment with `TITLE`, from `START_DATE` at `START_TIME` to `END_DATE` at `END_TIME`, with contact at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* `1, 2, 3, ...`.
-* `START_DATE` and `END_DATE` must be in the format `DD-MM-YYYY`.
-* `START_TIME` and `END_TIME` must be in the 24-hr format: HH:MM.
-* The new appointment will be listed on both the appointment list and the TeachConnect Calendar.
-****
+ ****
+ * This sets an appointment with `TITLE`, from `START_DATE` at `START_TIME` to `END_DATE` at `END_TIME`, with contact at the specified `INDEX`.
+ * The index refers to the index number shown in the most recent listing.
+ * The index *must be a positive integer* `1, 2, 3, ...`.
+ * `START_DATE` and `END_DATE` must be in the format `DD-MM-YYYY`.
+ * `START_TIME` and `END_TIME` must be in the 24-hr format: HH:MM.
+ * The new appointment will be listed on both the appointment list and the TeachConnect Calendar.
+ ****
 
-Examples:
+ Examples:
 
-* `set_appointment t/Tutoring session s/02-04-2018 19:00 e/02-04-2018 20:00` +
-Sets up an appointment on April 2nd, 2018, from 7pm to 8pm.
-+
-image::appointmentAdded.jpg[width="800"]
-    Figure 3: Appointment added successfully example
-+
+ * `set_appointment t/Tutoring session s/02-04-2018 19:00 e/02-04-2018 20:00` +
+ Sets up an appointment on April 2nd, 2018, from 7pm to 8pm.
+ +
+ image::appointmentAdded.jpg[width="800"]
+     Figure 3: Appointment added successfully example
+ +
 
```

Comment:
This change causes some display inconsistency with the Setting up an appointment part, so you should revert it.